src/transacting.md

```diff
@@ -281,6 +281,9 @@ curl -X POST "https://service.centrapay.com/payments/api/transactions.refund" \
 | 189        | 403       | {% break _ INSUFFICIENT_WALLET_BALANCE %}         | The wallet balance is less than the required amount                                                                                                                                                                                                       |
 | 190        | 200       | {% break _ TRANSACTION_ALREADY_EXISTS %}          | A successful payment transaction already exists for a payment request.                                                                                                                                                                                    |
 | 191        | 500       | {% break _ OPTIMISTIC_LOCK_ERROR %}               | A resource was updated concurrently. Request should be retried after refreshing latest state if applicable.                                                                                                                                               |
+| 195        | 400       | {% break - INVALID_ASSET_ID %}                    | The asset id is invalid                                                                                                                                                                                                                                   |
+| 196        | 400       | {% break - INVALID_WALLET_ADDRESS %}              | The wallet address is invalid                                                                                                                                                                                                                             |
+| 197        | 400       | {% break - INVALID_TRANSACTION %}                 | The transaction is invalid                                                                                                                                                                                                                                |
```

Comment:
The problem I have with these descriptions is that they don't add information. The asset id is invalid tells me just as much as INVALID_ASSET_ID

Could we reword this with examples? Is it that the formatting should be a particular way?

Wallet address invalid, does that mean the wallet doesn't exist or can't be found?

Invalid transaction, what kind of things fall into this camp?

Reply:
What do you think about the new descriptions?